README.md

```diff
@@ -2,41 +2,54 @@
 
 This repository contains work from "Badger Bytes" a group created within Sparta Global's **March '23 JAVA SDET** training course.
 
-This project involves testing the website and API <https://automationexercise.com/>, which has been specifically created for practising automation testing. The objective of the project is to analyze the website's functionality and create our own test cases for various API endpoints. By creating a testing framework we want to help future testers efficiently test the site using our framework.
+The objective of the project is to create test cases for: API endpoints and a website's functionality, creating a test framework for both parts. By creating this framework, we want to help other testers efficiently test the site using our framework. We tested the website and API <https://automationexercise.com/>, a site created for practising automation testing. The project was completed in two sprints: "API testing" and "web testing".
 
-## Main Objectives
+## Learning Objectives
```

Comment:
I think that probably this shouldn't say learning objectives. This is a project where we have been tasked with creating something for a potential client. Therefore the objective is probably to create rather than to learn.